src/extensions/sentry-integration.ts

```diff
@@ -97,6 +97,7 @@ export function createEventProcessor(
             // added manually to avoid any dependency on the lazily loaded content
             $exception_message: any
             $exception_type: any
```

Comment:
i haven't yet removed $exception_message from the sentry integration, but eventually yes will do.